Docs: how to use quilt_summarize for Perspective (#2748)
quiltdata/quilt: docs/Catalog/VisualizationDashboards.md

- `path` - file path relative to `quilt_summarize.json`
- `title` - title rendered instead of file path
- `description` - description in markdown format

> - `expanded` - expanded file view state (default is `false`)

Reviewer: I know this is a code change but I think expand is clearer (nice-to-have).
```suggestion
- `expand` - Display the file (`true`) or display a preview in an expandable box (`false`, default)
```

Author: https://github.com/quiltdata/quilt/pull/2783

to see the entire file contents.

In order to open the analysis and visualization capabilities of Perspecitve,

> click the vertical ellipsis, upper left. You can hold this settings control opened
> on page load in files showed with `quilt_summarize.json` by providing additional
> configuration:

Reviewer: ```suggestion
click the vertical ellipsis, upper left. To open the controls by default, set the `config.settings` property
in `quilt_summarize.json` as follows:
```

```json
// quilt_summarize.json
[
  {
    "path": "file1.csv",
    "types": [
      {
        "name": "perspective",
        "config": {
          "settings": true
        }
      }
    ]
  }
]
```

Besides `{ "settings": true }` you can provide any object previously saved from Perspective.

![](../imgs/perspective-save.png)

All filters and columns will be restored:

```json
// quilt_summarize.json
[
  {
    "path": "file1.csv",
    "types": [
      {
        "name": "perspective",

>         "config": {

Reviewer: Is .config *exactly* what gets passed to `.restore`? If so this is OK, otherwise we probably need to introduce a `config.restore` property here? Basically I want ot make sure the `settings` key is expected by perspective in a `restore` call.

Author: Yes, `config` value from `quilt_summarize.json` gets passed to `.restore()` method without changes.

`settings` is part of config, there is even note in documentation for `toggleConfig` method:
> This method is equivalent to viewer.restore({settings: force})

          "columns": ["name", "value"],
          "group_by": ["value"],
          "settings": true
        }
      }
    ]
  }
]
```

> Click, drag, and drop columns to explore the data.

Reviewer: Drag columns from the left to the
```suggestion
Drag columns from the left to the top to sort, filter, and pivot.
Use the menu upper left to try different visualizations.
Use the controls along the bottom to download, copy, resize, and more.
```
Below is a better and crushed image for you (shows more), you can replace perspective-save with this.
<img width="800" alt="perspective-controls" src="https://user-images.githubusercontent.com/1322715/162036945-4d3ff963-5e63-475f-8280-5c46f38751d4.png">
